scripts/lc-builds/toss4_clang.sh

```diff
@@ -7,18 +7,18 @@
 # SPDX-License-Identifier: (BSD-3-Clause)
 ###############################################################################
 
-if [[ $# -lt 1 ]]; then
+if [ "$1" == "" ]; then
```

Comment:
Why change this, it was consistent with other uses before?

Reply:
I think there's a mix of both checks. I'll take a pass through and make them all the same.

Reply:
All scripts are consistent now.